gpkit/tools/tools.py

```diff
@@ -43,7 +43,7 @@ def te_secant(var, nterm):
         Taylor expansion of secant(x), carried to nterm terms
     """
     # The first 12 Euler Numbers
-    E2n = np.asarray([1,
+    E2n = np.asarray([1.0,
```

Comment:
it was something marshall discovered, that this array became float-type only because of the large integers; I wanted to explicitly cast it.